### ISSUE TITLE
react-styleguidist does not recognize forwardRef components

### ISSUE THREAD

**Issue by jEnbuska** [styleguidist/react-styleguidist#1235]
**Current behavior**
react-styleguidist does not recognize React.forwardRef:s as components, and the examples representing forwardRef components do not get listed in the styleguidist page.

During build react-styleguidist prints this warning:
```
Warning: src/components/MyButton.tsx matches a pattern defined in “components” or “sections” options in your style guide config but doesn’t export a component.

It usually happens when using third-party libraries, see possible solutions here:
https://react-styleguidist.js.org/docs/thirdparties.html
```

**To reproduce**

Create forwardRef button:
```
export const MyButton = React.forwardRef((props, ref) => {
  return <button  {...props} ref={ref} className={addClass('my-button', props.className)} />
});
```
Create example and add MyButton to styleguide.config.js sections

```
{
                    name: 'Components',
                    content: 'docs/empty.md',
                    components:
                        [
                            'src/components/Paper.tsx',
                            'src/components/Input.tsx',
                            'src/components/Button.tsx',
                            ...
                        ]
 },
...
```

**Comment by iamtyce** [comment]
Experiencing a similar issue where props and methods are not being rendered for components with `forwardRef`

**Comment by iamtyce** [comment]
Can confirm that updating to `react-styleguidist@^8.0.0` and `webpack@^4.0.0` fixed this for me (was on previous majors for both)
